LISA/SqlDataStorage.cpp

```diff
@@ -185,6 +264,99 @@ namespace LISA {
         return paths;
     }
 
+    void SqlDataStorage::InsertIntoApps(const std::string& type,
+                                        const std::string& id,
+                                        const std::string& appPath,
+                                        const std::string& timeCreated)
+    {
+        std::stringstream query;
+        query << "INSERT INTO apps VALUES(NULL, ";
+        query << "'" << type << "', ";
```

Comment:
Please remember to change it to use prepared statements once other queries are reworked to use that solution